nyaa/search.py

```diff
@@ -3,23 +3,115 @@
 import math
 import json
 import shlex
+from datetime import datetime, timedelta
 
 from nyaa import app, db
-from nyaa import models
+from nyaa import models, torrents
 
 import sqlalchemy_fulltext.modes as FullTextMode
 from sqlalchemy_fulltext import FullTextSearch
 from elasticsearch import Elasticsearch
 from elasticsearch_dsl import Search, Q
 
+class ElasticTorrentStatsAccessor:    
```

Comment:
Can it? it's a nested property: `torrent.stats.download_count`.
And if yes, wouldn't that be quite the hack? since getattr is already being used to forward property accesses to `torrent`.
Please note that the exposed property is `torrent.stats.download_count` while elastic has it in `elastic.download_count`.